dev-python/pytest-pep8/pytest-pep8-1.0.6-r2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="pytest plugin to check PEP8 requirements"
+HOMEPAGE="https://pypi.org/project/pytest-pep8/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/pep8-1.7.1[${PYTHON_USEDEP}]
+	>=dev-python/pytest-4.6.9[${PYTHON_USEDEP}]
+	>=dev-python/pytest-cache-1.0-r2[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/1.0.6-MANIFEST.patch
+)
+
+python_test() {
+	${EPYTHON} test_pep8.py || die
+}
```

Comment:
See if `distutils_enable_tests pytest` works here rather than the custom script, often those wrappers aren't needed. (the pytest dep can be dropped if this is the case, though the other test deps should be kept)

Otherwise, please do `|| die "Tests fail with ${EPYTHON}"

Reply:
didn't work